cmd/serverless-init/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/initcontainer"
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/log"
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/metadata"
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/metric"
+	"github.com/DataDog/datadog-agent/cmd/serverless-init/tag"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/serverless/metrics"
+	"github.com/DataDog/datadog-agent/pkg/serverless/trace"
+)
+
+const (
+	datadogConfigPath = "/var/task/datadog.yaml"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		panic("invalid arguments")
```

Comment:
Is it possible for a user to misconfigure their setup, and see this error message? Should it be more descriptive?